spec/features/meters/1_user_views_index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+feature 'view meters#index', %Q{
+  As a Guest
+  I want to view the meters#index
+  So I can pick from all meters.
+  } do
+  # Acceptance
+  # When a guest visits the meters#index_path,
+  # they should expect to see a list of all meters
+  # The list should be sorted alphabetically by Title
+  # The list should include the Name
+
+  let!(:meter1) { FactoryGirl.create(:meter, name: "5/8") }
+  let!(:meter2) { FactoryGirl.create(:meter, name: "3/4") }
+  let!(:meter3) { FactoryGirl.create(:meter, name: "8/16") }
+
+  pending 'sucessfully view the meters#index' do
+    visit meters_path
+
+    meters = page.all("div#meterlist ul li")
+
+    expect(meters[0]).to have_content(meter1.name)
+    expect(meters[1]).to have_content(meter3.name)
+    expect(meters[2]).to have_content(meter2.name)
+  end
```

Comment:
If you are not necessarily testing for order, another thing to consider is capybara's `within` method / construct for this pending test.